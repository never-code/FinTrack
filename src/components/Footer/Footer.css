.footer-container {
  padding: 2%;
  font-weight: 900;
  text-align: center;
  letter-spacing: 3px;
  background-color: #1f5156;
}
a {
  padding: 10px;
  color: black;
}
a:hover {
  color: whitesmoke;
}
.linkss {
  font-size: 25px;
}
.follow {
  color: black;
}
