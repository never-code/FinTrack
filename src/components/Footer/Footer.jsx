import React from 'react';
import './Footer.scss';
const Footer = () => (
  <div className="footer-container">
    <div className="linkss">
      <a href="https://github.com/never-code"> <i class="fab fa-github"></i> </a>
      
      </div>
  </div>
);

export default Footer;
