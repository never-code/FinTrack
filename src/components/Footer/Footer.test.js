import React from 'react';
import { render } from '@testing-library/react';
import Footer from './Footer';

describe('Header ', () => {
  it('should render successfully', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
