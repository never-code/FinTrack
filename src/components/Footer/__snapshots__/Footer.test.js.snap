// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header  should render successfully 1`] = `
<div>
  <div
    class="footer-container"
  >
    Made by
    <a
      href="https://suhas010.com"
      rel="noreferrer"
      target="_blank"
    >
      Suhas010
    </a>
    ðŸ¤“
    <div
      class="follow"
    >
      <a
        href="https://twitter.com/suhas0101"
        rel="noreferrer"
        target="_blank"
      >
        Twitter
      </a>
      <a
        href="https://linkedin.com/in/suhas-more"
        rel="noreferrer"
        target="_blank"
      >
        LinkeIn
      </a>
      <a
        href="https://github.com/suhas010"
        rel="noreferrer"
        target="_blank"
      >
        GitHub
      </a>
    </div>
  </div>
</div>
`;
